src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -230,6 +230,18 @@ public static class ConfigurationKeys
         /// </summary>
         public const string DiagnosticSourceEnabled = "DD_DIAGNOSTIC_SOURCE_ENABLED";
 
+        /// <summary>
+        /// Configuration key for the application's server http statuses to set spans as errors by.
+        /// </summary>
+        /// <seealso cref="TracerSettings.HttpServerErrorCodes"/>
+        public const string HttpServerErrorCodes = "DD_HTTP_SERVER_ERROR_STATUSES";
```

Comment:
The field `HttpServerErrorCodes` should be called `HttpServerErrorStatuses`

Reply:
Hah. Maximo originally had `HttpServerErrors` and I asked him to change to either `HttpServerErrorCodes` or `HttpServerErrorStatuses`. I may have also said that I preferred `...Codes`. https://github.com/DataDog/dd-trace-dotnet/pull/1080#discussion_r540430369

Reply:
`HttpServerErrorStatusCodes`/`HttpClientErrorStatusCodes` are another possibility.

I wouldn't worry about matching the (unfortunately named) `DD_HTTP_SERVER_ERROR_STATUSES` closely. For C# identifiers, we consider having an idiomatic name more important than matching the cross-language environment variable name. Other examples that don't match:
- `DD_SERVICE`: `ServiceName` instead of `Service`
- `DD_TRACE_DEBUG`: `DebugEnabled` instead of `Debug`
- `DD_TRACE_AGENT_URL`: `AgentUri` instead of `AgentUrl`

Reply:
I like `HttpServerErrorStatusCodes`/ `HttpClientErrorStatusCodes` 😀